Update convert-unmanaged-to-managed-disks.md (#111884)
Azure/azure-content: articles/virtual-machines/windows/convert-unmanaged-to-managed-disks.md

* Review [the FAQ about migration to Managed Disks](../faq-for-disks.yml).

> * Before converting, make sure all the VM extensions are in the 'Provisioning succeeded' state or the migration will fail.

Reviewer: ```suggestion
* Ensure the VM is in a healthy sate before converting.

```
